packages/web3-eth/src/web3_eth.ts

```diff
@@ -167,6 +327,68 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 		return rpcMethodsWrappers.getCode(this, address, blockNumber, returnFormat);
 	}
 
+	/**
+	 * @param block The {@link BlockNumberOrTag} (defaults to {@link Web3Eth.defaultBlock}) or block hash of the desired block.
```

Comment:
its desc :

```suggestion
	 * Returns a block matching the block number, block hash or block tag.
	 *
	 * @param block The {@link BlockNumberOrTag} (defaults to {@link Web3Eth.defaultBlock}) or block hash of the desired block.
```

Reply:
`block` is the name of the first parameter of this method, referring to the block identifier that's being requested

Reply:
thats correct. I suggested to add description before block param.

Reply:
Implemented via [this](https://github.com/ChainSafe/web3.js/pull/5251/commits/3ce68c0082d4838db6489bc0384ca04652293184) commit